clojure-mode.el

```diff
@@ -195,6 +195,11 @@ Out-of-the box clojure-mode understands lein, boot and gradle."
   (let ((map (make-sparse-keymap)))
     (define-key map (kbd "C-:") #'clojure-toggle-keyword-string)
     (define-key map (kbd "C-c SPC") #'clojure-align)
+    (define-key map (kbd "C-c th") #'clojure-thread)
```

Comment:
We should change the defaults to something more common - e.g. `C-c C-r` something.